BlueprintUI/Sources/Extensions/UIAccessibilityTraits+Bueprint.swift

```diff
@@ -20,12 +20,12 @@ extension UIAccessibilityTraits {
     /// (lldb) po String(134217728, radix: 2).count - 1
     /// 27
     /// ```
-    public static let backButton: UIAccessibilityTraits = UIAccessibilityTraits(rawValue: 1 << 27)
+    package static let backButton: UIAccessibilityTraits = UIAccessibilityTraits(rawValue: 1 << 27)
 
 
     /// `UIAccessibilityTraits` contains a trait that is used on `UISwitch`
     /// prior to iOS 17 this trait was private but was later exposed as `.toggleButton`.
     /// We will reference the trait by its raw value but provide an alternative name to avoid conflicting with the public trait .
     ///
-    public static let _toggleButton = UIAccessibilityTraits(rawValue: 1 << 53)
+    package static let _toggleButton = UIAccessibilityTraits(rawValue: 1 << 53)
```

Comment:
'backButton' is inaccessible due to 'package' protection level
	'_toggleButton' is inaccessible due to 'package' protection level